pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -0,0 +1,145 @@
+'''
+
+[context.cmd.example]
+one='''
+## Set the current context
+$ rhoas context use --name my-context
+
+## List contexts
+$ rhoas context list
+'''
+
+[context.use.cmd]
+
+[context.use.cmd.shortDescription]
+one='Set the current context'
+
+[context.use.cmd.longDescription]
+one='''
+Select a context to be used as the current context.
+When you set the context to be used, it is set as the current context for all service based commands.
+'''
+
+[context.use.cmd.example]
+one='''
+## Set the current context
+$ rhoas context use --name my-context
+'''
+
+[context.use.successMessage]
+one='Current context set to "{{.Name}}"'
+
+[context.status.cmd]
+
+[context.status.cmd.shortDescription]
+one='Show status of the context'
+
+[context.status.cmd.longDescription]
+one='''
+Print overview of the services in associated with a user context
+'''
+
+[context.status.cmd.example]
+one='''
+## Show status of current context
+$ rhoas context status
+
+## Show status of specified context
+$ rhoas context status --name my-context
+'''
+
+[context.list.cmd]
+
+[context.list.cmd.shortDescription]
+one='List contexts'
+
+[context.list.cmd.longDescription]
+one='List currently available user contexts'
+
+[context.list.cmd.example]
+one='''
+## List contexts
+$ rhoas context list
+'''
+
+[context.list.log.noContexts]
+one='''
+No user contexts exist.
+
+Run the following command to create an user context:
+
+  rhoas context create
+'''
+
+[context.create.cmd]
+
+[context.create.cmd.shortDescription]
+one='Create a user context'
+
+[context.create.cmd.longDescription]
+one='Create a user context and assign associated service identifiers'
+
+[context.create.cmd.example]
+one='''
+## Create context with Kafka ID and Service Registry ID
+$ rhoas context create --name my-context --kafka-id c8696ncpoj7gdjmaiqog --registry-id 0282d488-52b3-405b-9e30-9f6f9407de57
+'''
+
+[context.create.log.successMessage]
+one='Context created successfully'
+
+[context.create.log.alreadyExists]
+one='Context with name "{{.Name}}" already exists'
+
+[context.common.flag.name]
+one='Name of the context'
+
+[context.common.error.noRegistryID]
+one='''
+context doesn't have a Service Registry ID.
+Use `rhoas context status` commands to review your context
+'''
+
+[context.common.error.noKafkaID]
+one='''
+context doesn't have a Kafka Instance ID.
+Use `rhoas context status` commands to review your context
+'''
+
+[context.common.error.registry.notFound]
+one='''
+Service registry instance doesnt exist, you need to review your context using command `rhoas context status`
+'''
+
+[context.common.error.kafka.notFound]
+one='''
+Kafka instance doesnt exist, you need to review your context using command `rhoas context status`
+'''
+
+[context.common.error.context.notFound]
+one='''
+context with name "{{.Name}}" does not exist
+
+Run `rhoas context list` to view available contexts
+'''
+
+[context.common.error.notSet]
+one='''
+context has not been set. Run the following command to set context:
```

Comment:
This message is very important to establish good UX and extend this message little bit later.